modules/multi-architecture-creating-x86_64-bootimage.adoc

```diff
@@ -0,0 +1,137 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-creating-x86-64-bootimage_{context}"]
+= Creating a 64-bit x86 boot image using the Azure image gallery
+
+The following procedure describes how to manually generate a 64-bit x86 boot image.
+
+.Prerequisites
+
+* You installed the Azure CLI (`az`).
+* You created a single-architecture Azure installer-provisioned cluster with the multi-architecture installer binary.
+
+.Procedure
+
+. Log in to your Azure account by running the following command:
++
+[source,terminal]
+----
+$ az login
+----
+. Create a storage account and upload the `x86_64` virtual hard disk (VHD) to your storage account. The {product-title} installation program creates a resource group. However, the boot image can also be uploaded to a custom named resource group:
```

Comment:
I think this step still needs to mention running a command, even if it's not the last sentence in the step:
```suggestion
. Create a storage account and upload the `x86_64` virtual hard disk (VHD) to your storage account by running the following command. The {product-title} installation program creates a resource group. However, the boot image can also be uploaded to a custom named resource group:
```

Reply:
Updated as per the suggestion.